### ISSUE TITLE
Description of first Alert takes precedence, additional descriptions aren't shown.

### ISSUE THREAD

**Issue by mweinelt** [freifunk-darmstadt/go-prom-irc#4]
### Idea

If the description is the same among all alerts
```
  - host1, host2, host3
```

If the description is different
```
  - host1, host3: 11 processes are running on outdated libs
  - host2: 19 processes are running on outdated libs
```

### Log

```
23:56 <ffda-prom> [RESOLVED:2] CheckRestartProcessCount - 11 processes are running on outdated libs
23:56 <ffda-prom>     forum, netbox
23:56 <ffda-prom> [FIRING:2] CheckRestartProcessCount - 11 processes are running on outdated libs
23:56 <ffda-prom>     forum, netbox
```

```json
{
  "version": "3",
  "groupKey": 5271911728668818000,
  "status": "firing",
  "receiver": "irc",
  "groupLabels": {
    "alertname": "CheckRestartProcessCount"
  },
  "commonLabels": {
    "alertname": "CheckRestartProcessCount",
    "job": "node",
    "severity": "warning"
  },
  "commonAnnotations": {},
  "externalURL": "https://prometheus.darmstadt.freifunk.net/alertmanager",
  "alerts": [
    {
      "labels": {
        "alertname": "CheckRestartProcessCount",
        "instance": "netbox.darmstadt.freifunk.net:9100",
        "job": "node",
        "severity": "warning"
      },
      "annotations": {
        "description": "11 processes are running on outdated libs",
        "summary": "Instance netbox.darmstadt.freifunk.net:9100: There are 11 processes with old libs."
      },
      "startsAt": "2017-11-04T23:32:37.803+01:00",
      "endsAt": "0001-01-01T00:00:00Z"
    },
    {
      "labels": {
        "alertname": "CheckRestartProcessCount",
        "instance": "forum.darmstadt.freifunk.net:9100",
        "job": "node",
        "severity": "warning"
      },
      "annotations": {
        "description": "19 processes are running on outdated libs",
        "summary": "Instance forum.darmstadt.freifunk.net:9100: There are 19 processes with old libs."
      },
      "startsAt": "2017-11-04T23:32:37.803+01:00",
      "endsAt": "0001-01-01T00:00:00Z"
    }
  ]
}
```

**Comment by fleaz** [comment]
Because we can only show one line of text for all hosts, we should agree on using the "description" or some other of the annotations as a "generic" description of the alert.
e.g. something like this:
```
      "annotations": {
        "description": "Number of processes with outdated libs",
        "summary": "netbox.darmstadt.freifunk.net: There are 11 processes with old libs."
      },
```

**Comment by fleaz** [comment]
Because we can't 'fix' this in go-prom-irc, it should be written in the README so everyone knows how to adapt their descriptions/summarys accordingly

**Comment by mweinelt** [comment]
There could be a verbose message, like:

```
- host1 (label1: value, label2: value, label3: value)
- host2 (label1: value, label2: value, label3: value)
```
Maybe for labels prefixed with `irc_`

**Comment by fleaz** [comment]
This should be fixed by a combination of  f4d0f428861f9eeccf7f20dfb6f18719aca8fe6a providing more values with the alertmanager.
